crates/cac_client/src/lib.rs

```diff
@@ -166,10 +165,10 @@ impl Client {
                     .as_str()
                     .ok_or_else(|| {
                         log::error!("Prefix is not a valid string.");
-                        format!("Prefix is not a valid string.")
```

Comment:
Do we avoid the macro here to reduce compile time? Whats the benefit of changing format! to to_string?